specification/_types/sort.ts

```diff
@@ -96,7 +98,12 @@ export type SortCombinations = Field | SortOptions
 
 export type Sort = SortCombinations | SortCombinations[]
 
-export type SortResults = Array<long | double | string | null>
+/**
+ * Only used when sorting by "_tsid"
+ */
+export type SortTsid = Dictionary<Field, FieldValue>
+
+export type SortResults = Array<long | double | string | null | SortTsid>
```

Comment:
We should change this to `Array<FieldValue>`, because this is what it actually is, and `any` was recently added to `FieldValue`.